torchx/runtime/container/Dockerfile

```diff
@@ -2,27 +2,12 @@ FROM pytorch/pytorch:1.9.0-cuda10.2-cudnn7-runtime
 
```

Comment:
this would change to `pytorch/pytorch:1.10-cuda10.2-cudnn7-runtime` once that's out on dockerhub.